readthedocs/builds/views.py

```diff
@@ -33,7 +34,24 @@ def get_queryset(self):
         return queryset
 
 
-class BuildList(BuildBase, ListView):
+class BuildTriggerMixin(object):
+    def post(self, request, project_slug):
```

Comment:
This should not be only for logged in user?

Reply:
`login_required` decorator can be used here

Reply:
Good point, we do need to check for a proper `request.user` here.

Reply:
> we do need to check for a proper request.user here

what do you mean with this?